src/layouts/components/HeaderNavLinkItem.tsx

```diff
@@ -0,0 +1,97 @@
+import { DFIIcon } from "@public/assets/icon/DFIIcon";
+import { WalletIcon } from "@public/assets/icon/WalletIcon";
+import { GitHubIcon } from "@public/assets/icon/GitHubIcon";
+import { InvestIcon } from "@public/assets/icon/InvestIcon";
+import { MasternodeIcon } from "@public/assets/icon/MasternodeIcon";
+import { DexIcon } from "@public/assets/icon/DexIcon";
+import classNames from "classnames";
+import { GovernanceIcon } from "@public/assets/icon/GovernanceIcon";
+import { AppsIcon } from "@public/assets/icon/AppsIcon";
+import { JellyfishIcon } from "@public/assets/icon/JellyfishIcon";
+import { WhitePaperIcon } from "@public/assets/icon/WhitePaperIcon";
+import { SecurityIcon } from "@public/assets/icon/SecurityIcon";
+import { MediaAssetsIcon } from "@public/assets/icon/MediaAssetsIcon";
+import { DeveloperResourceIcon } from "@public/assets/icon/DeveloperResourceIcon";
+import { Dispatch, SetStateAction } from "react";
+
+export function HeaderNavLinkItem({
+  icon,
+  label,
+  subLabel,
+  href,
+  hoverState,
+  setHoverState,
+}: {
+  icon?: string;
+  label: string;
+  subLabel?: string;
+  href: string;
+  hoverState: string | undefined;
+  setHoverState: Dispatch<SetStateAction<string | undefined>>;
+}) {
+  const Icon = iconMapping[icon] as React.ElementType;
+  const iconsStrokes = [
+    "masternode",
+    "invest",
+    "apps",
+    "github",
+    "whitepaper",
+    "media",
+    "security",
+    "developer",
+  ];
+  return (
+    <a
+      onMouseEnter={() => {
+        setHoverState(label);
+      }}
+      onMouseLeave={() => {
+        setHoverState(undefined);
+      }}
+      href={href}
+      className={classNames(
+        "group flex flex-row items-center gap-x-[28px]",
+        hoverState !== undefined && hoverState !== label
```

Comment:
if you check the prototype on figma (the interactive one), when you hover on one menu, only that menu should change in UI (color), all others shall maintain the original UI
<img width="770" alt="image" src="https://user-images.githubusercontent.com/40191153/221066265-16d389c1-0b7a-42a5-b371-8262ffae4fc7.png">